Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>OPENAI_API_KEY=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\r\nHUGGINGFACEHUB_API_TOKEN=\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	
+++ b/.env	
@@ -1,2 +1,2 @@
-OPENAI_API_KEY=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
+OPENAI_API_KEY="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"
 HUGGINGFACEHUB_API_TOKEN=
